### PR TITLE
Make `convert` available without import

### PR DESCRIPTION
**Pull request by rjolly**

### CODE REVIEW

File: tests/pos/convert.scala [hunk 1]
```diff
@@ -1,7 +1,5 @@
-import Conversion.convert
-
 given Conversion[String, Int] = _.length
 given Conversion[Int, String] = _.toString
 
 def f(x: String): Int = x.convert
-def g(x: Int): String = x.convert[String]
+def g(x: Int): String = x.convert
```
Comment (sjrd) [inline tests/pos/convert.scala:5]: This won't work if there are two `Conversion[Int, Y]` in scope for two different `Y`s.

Comment (rjolly) [reply tests/pos/convert.scala:8]: It does with a suitable type ascription, see updated test.

Comment (odersky) [reply tests/pos/convert.scala:8]: @sjrd Looks like @rjolly is right. Do you agree?

**Review by odersky — APPROVED** [review]
I think this better than what we had.

File: tests/pos/convert.scala (hunk 1 above)
Comment (sjrd) [reply tests/pos/convert.scala:8]: Yes, apparently it works, indeed.

**Review by sjrd — CHANGES_REQUESTED** [review]
Changing the name to `~` is a different battle, and should definitely not be part of this PR.

**Review by sjrd — APPROVED** [review]

**Pull request merged**